addon/components/step-manager.js

```diff
@@ -226,6 +227,24 @@ export default Component.extend({
       const to = get(this, 'transitions').peek();
 
       this.send('transition-to-step', to, value);
+    },
+
+    /**
+     * Transition to the "previous" step
+     *
+     * When called, this action will go back to the previous step according to
+     * the step which was visited before entering the currentStep
+     *
+     * The first step will not transition to anything.
+     *
+     * @method transition-to-previous-step
+     * @param {*} value the value to pass to the transition actions
+     * @public
+     */
+    'transition-to-previous-step'(value) {
+      const to = get(this, '_lastStep');
+
+      if (to) this.send('transition-to-step', to, value);
```

Comment:
Style nit-pick: can you put this inside a block, rather than putting the condition inline with the action?